package.json

```diff
@@ -1,14 +1,11 @@
 {
   "name": "seng-config",
-  "version": "1.0.2",
+  "version": "1.1.0",
   "description": "Manages a config file by merging environments and providing a useful API",
   "main": "dist/index.js",
-  "typings": "dist/index",
   "scripts": {
-    "postinstall": "npm run typings",
     "prepublish": "npm run build",
-    "typings": "typings install",
-    "build": "rm -rf dist && tsc -p ./",
+    "build": "rimraf -rf dist && tsc -p ./",
```

Comment:
Can you change this to use shx instead?
https://github.com/shelljs/shx
Adds support for more commands besides "rm", and we also use this in other seng projects.

(don't forget to also update docs/changelog)